LICENSE

```diff
@@ -1,28 +1,25 @@
-﻿Copyright (c) 2011-2016, Humanoid Lab, Georgia Tech Research Corporation
+Copyright (c) 2011-2016, Humanoid Lab, Georgia Tech Research Corporation
```

Comment:
I'm curious: What actually changed here? Some invisible character?

Reply:
Yeah, although it's not visible here there was an unknown (to me) character at the beginning of the line. 🤔